HealthcareSystem/PatientWebApp/Controllers/ExaminationController.cs

```diff
@@ -178,6 +196,8 @@ public ActionResult GetFollowingExaminationsByPatient(string patientJmbg)
         /// </summary>
         /// <param name="id">examination id</param>
         /// <returns>if alright returns code 200(Ok), if returned value is null returns 404, if connection lost returns 500</returns>
+        /// 
+        [Authorize(Roles = UserRoles.Patient)]
         [HttpGet("{id}")]
         public IActionResult GetExaminationById(int id)
```

Comment:
Jmbg check here as well after you fetch the examination, it needs to be for this patient.

Reply:
Fixed.